test/units/TestFilters.py

```diff
@@ -183,3 +184,23 @@ def test_min(self):
     def test_max(self):
         a = ansible.runner.filter_plugins.core.max([3, 2, 5, 4])
         assert a == 5
+        
```

Comment:
this is a very old version of this file, which does not even exist in current devel